src/bitcoin-tx.cpp

```diff
@@ -603,17 +603,18 @@ static void MutateTxSign(CMutableTransaction& tx, const std::string& flagStr)
         const CScript& prevPubKey = coins->vout[txin.prevout.n].scriptPubKey;
         const CAmount& amount = coins->vout[txin.prevout.n].nValue;
 
+	//TODO: This is obviously wrong, need to fix later
+	const CTransactionRef& coinbaseTx = MakeTransactionRef(tx);
```

Comment:
Not sure what the best way to fix this is. Should we just add a parameter to `MutateTxSign()` to pass the coinbase transaction that commits to the critical hash for the `OP_BRIBEVERIFY` output we are spending?